k2/python/csrc/torch/ragged.cu

```diff
@@ -0,0 +1,95 @@
+/**
+ * @brief python wrappers for Ragged<T>.
+ *
+ * @copyright
+ * Copyright (c)  2020  Mobvoi Inc.        (authors: Fangjun Kuang)
+ *
+ * @copyright
+ * See LICENSE for clarification regarding multiple authors
+ */
+
+#include <vector>
+
+#include "k2/csrc/ragged.h"
+#include "k2/python/csrc/torch/ragged.h"
+#include "k2/python/csrc/torch/torch_util.h"
+#include "torch/extension.h"
+
+namespace k2 {
+
+static void PybindRaggedShape(py::module &m) {
+  using PyClass = RaggedShape;
+  py::class_<PyClass> pyclass(m, "RaggedShape");
+
+  // NOTE: we do not provide constructors of RaggedShape
+  // in Python, so the only way to get an instance of
+  // RaggedShape is from Ragged<T>.
+
+  pyclass.def("dim0", &PyClass::Dim0);
+  pyclass.def("total_size", &PyClass::TotSize, py::arg("axis"));
+
+  // TODO(fangjun): Append has not been implemented yet
+  // pyclass.def("append", &PyClass::Append, py::arg("other"));
+
+  pyclass.def("numel", &PyClass::NumElements);
```

Comment:
We'll need to document the Python interface somehow, esp. as regards these naming differences.
One possibility is to document this by the C++ declarations, i.e. make a note of their names in python?

Reply:
I agree. I have been planning to add documentation to every wrapped API.
What is the progress of setting up the sphinx documentation environment for K2 @songmeixu ?